zyte_common_items/processors.py

```diff
@@ -330,6 +363,45 @@ def aggregateRating(self):
     return value
 
 
+def images_processor(value: Any, page: Any) -> Any:
+    """Convert the data into a list of :class:`~zyte_common_items.Image`
+    objects if possible.
+
+    If the input is a string, it's used as a url for returning image object.
+
+    If input is either an iterable of strings or mappings with "url" key, they are
+    used to populate image objects.
+
+    Other inputs are returned unchanged.
+    """
+
+    value = _handle_selectorlist(value)
```

Comment:
Why is this needed? It seems that if a SelectorList is passed, the first item is taken, and then returned as-is. But it doesn't seem to be really supported, as in the end it's not a valid Image instance (as you have in TODO). So, it seems for now it can be dropped.

Reply:
Moved it inside `TODO` comment in d2ab672afffdc9aad5a064a8345e663d4af8dd33 since indeed it isn't needed for now.